test/test_tree_3d_mhd.jl

```diff
@@ -291,6 +291,20 @@ end
         @test (@allocated Trixi.rhs!(du_ode, u_ode, semi, t)) < 1000
     end
 end
+
+  @trixi_testset "elixir_mhd_diffusive_alfven_wave.jl" begin
+    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_mhd_diffusive_alfven_wave.jl"),
+      l2 = [3.470701487566952e-5, 0.00037320266655784104, 0.0001814791326953306, 0.0004187613798734842, 0.00011587163989250743, 0.00037032036399761947, 0.00018270779781144185, 0.00041229458967006154, 2.7753210639128732e-5],
+      linf = [0.0001348833811495309, 0.000892691474205648, 0.00043846917904940587, 0.0010014206836504308, 0.000443288547495424, 0.0008149498186467707, 0.00042111533550182045, 0.000918156282098577, 0.00010109055017304109])
+  end
+
+  @trixi_testset "elixir_mhd_dynamo.jl" begin
+    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_mhd_dynamo.jl"),
+      l2   = [5.136817044406367e-9, 1.0728023104350635e-6, 1.0741472192876857e-6, 1.1384862784905957e-6, 8.505678187506508e-8, 1.2404355188990195e-8, 1.2260968462092833e-8, 1.2453935882908254e-8, 5.406090684284657e-9],
+      linf = [6.223255566428776e-8, 3.476280403710667e-6, 3.2024943277794614e-6, 3.2985189119007034e-6, 1.0206843477078564e-6, 7.224815773523103e-8, 9.16975614740533e-8, 7.92452544029745e-8, 1.201338384265285e-7],
+     tspan = (0.0, 0.001))
+  end
```

Comment:
Here we definitely should check for allocations as for the other tests.

Reply:
I removed the MHD dynamo test. Dynamos require a long time to be excited and become fully turbulent. Since we already test the visco-resistive equations in [test/test_tree_3d_mhd.jl](https://github.com/trixi-framework/Trixi.jl/pull/1323/files/d3c90f6208dea269d871650feb0131b503646188#diff-6abf3de2eaa2fb7d2d53a57d2fa307cdf728cb9949862d478f1597dae5ee14d1), we don't need this.

Reply:
Hm not so sure if the others will accept this, usually every elixir has a test.

Reply:
But maybe reduce the `tspan` such that the test does not run longer than 10 sec.

Reply:
The dynamo example elixir is rather specific. So, I decided to remove it. The Alfven wave example should be sufficient to show case the usage of the visco-resistive equations.